frontend/src/components/mine/Permit/MinePermitCard.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import PropTypes from "prop-types";
 import CustomPropTypes from "@/customPropTypes";
-import { formatTitleString } from "@/utils/helpers";
+import { formatTitleString, formatDate } from "@/utils/helpers";
```

Comment:
`formatDate` seems kinda generic... should the method name be more verbose? Just a thought

Reply:
Not my favorite, but Ryan did a good job explaining it to me. To be reviewed later.